src/content/tools/vs-code.md

```diff
@@ -4,19 +4,49 @@ description: You can use VS Code to develop Dart apps.
 toc: false
 ---
 
-With the Dart plugin, you can use [Visual Studio Code (VS Code)][vs-code]
-to develop Dart apps.
-The Flutter site has details on how to set up and use VS Code for Flutter apps.
+To develop Dart apps,
+install [Visual Studio Code][vs-code] (VS Code)
+and add the [Dart extension][].
 
-Until this page has more information,
-see the Flutter documentation for instructions that you can apply to
-other kinds of Dart apps:
+## Install VS Code
 
-* [Set up an editor][setup] on the Flutter site has
-  instructions for installing the Flutter and Dart plugins into VS Code.
-* [Visual Studio Code][vscode-flutter] on the Flutter site has
-  details on using VS Code to develop Flutter apps.
+To build and debug apps,
+you can use the [VS Code][] integrated development environment.
 
-[setup]: {{site.flutter-docs}}/get-started/editor?tab=vscode
+To install the latest version of VS Code,
+follow Microsoft's instructions for the relevant platform:
+
+- [Install on macOS][]
+- [Install on Windows][]
+- [Install on Linux][]
+
+[VS Code]: https://code.visualstudio.com/
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Dart extension
+
+With the Dart extension installed,
+you can compile, deploy, and debug Dart apps.
```

Comment:
Again, seems like another unnecessary repetition of the intro sentence lines 7-9. Instead I'd start this section something like:

```suggestion
The Dart extension for VS Code enables support for
Dart and Flutter specific development,
like syntax highlighting, package resolution, hot reload, and more.
For the full list of features,
see the [extension documentation][Dart extension]
```

Also, you rewrote the original lines 7-9 that started with "With the Dart plugin...", but then used that same structure here ("With the Dart extension..."). Is starting a sentence like that something that needs to be rewritten or not?

Reply:
Eh, it could go either way.